inginious/frontend/user_manager.py

```diff
@@ -417,6 +417,11 @@ def get_user_api_key(self, username, create=True):
             apikey = retval.get("apikey", None)
         return apikey
 
+    def get_user_custom_values(self, username, courseid):
```

Comment:
It's probably worth using this newly defined function everywhere its content is duplicated. See:

https://github.com/UCL-INGI/INGInious/pull/835/files#diff-9aa5cd905c7c870607c364baaf99f4d28a5c5330d4b8388fa838d268ec86a6b5R22-R25

https://github.com/UCL-INGI/INGInious/pull/835/files#diff-9aa5cd905c7c870607c364baaf99f4d28a5c5330d4b8388fa838d268ec86a6b5R31-R34